client/extensions/woocommerce/app/products/product-form.js

```diff
@@ -2,15 +2,12 @@
  * External dependencies
  */
 import React, { Component, PropTypes } from 'react';
-import i18n from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-import FoldableCard from 'components/foldable-card';
 import ProductFormDetailsCard from './product-form-details-card';
-import ProductVariationTypesForm from './product-variation-types-form';
-import FormToggle from 'components/forms/form-toggle';
+import ProductFormVariationCard from './product-form-variation-card';
```

Comment:
This is a nit, but would you consider having it be plural like the details card, i.e. `ProductFormVariationsCard`